FogCreek/Glitch-Community
# different-print: Change "delete project" animation (and test it?)

## Links
* Remix link: https://different-print.glitch.me/
* Manuscript link: https://glitch.manuscript.com/f/cases/assign/3328210/Deleting-a-project-then-deleting-another-can-be-confusing
* Specs/Notion docs: N/A

## GIF/Screenshots:

## Changes:
* The delete animation for a project is now a partial fade, rather than sliding down

## How To Test:
* You can go to your profile, delete a project, and watch the animation
- OR -
* You can try running `./sh/cypress.sh different-print` and it should run through the scenario for you (running it against `community` will show you how it looks on that remix)

## Feedback I'm looking for:
- If you're at all familiar with Cypress I would like another set of eyes on some of the testing and test setup.
- How does the delete animation look?

## Things left to do before deploying:
- [ ] IDK maybe remove my test account information, maybe not?
- [ ] Clean up some of the test setup probably.
- [ ] Decide if we like the animation change.
